src/routes/subscriptions.js

```diff
@@ -0,0 +1,83 @@
+const { StatusCodes } = require('http-status-codes');
+
+const express = require('express');
+const router = express.Router();
+const db = require('../storage/DataAccess');
+const COLLECTION = 'plandefs';
```

Comment:
If we're storing actual FHIR resources in the db, I'd suggest we use the full resource name as the collection name for clarity & consistency